docs/data/joy/customization/creating-themed-components/StatFullTemplate.tsx

```diff
@@ -16,18 +16,25 @@ interface StatOwnerState extends StatProps {
 const StatRoot = styled('div', {
   name: 'JoyStat',
   slot: 'root',
-})<{ ownerState: StatOwnerState }>(({ theme, ownerState }) => ({
+})<{ ownerState: StatOwnerState }>(({ theme }) => ({
```

Comment:
We could have dropped the type

Reply:
I don't think so, the type should be there. It's for the consumer of the component, not the codemod to decide.